common/src/main/java/net/william278/huskchat/discord/DiscordMessageFormat.java

```diff
@@ -0,0 +1,55 @@
+package net.william278.huskchat.discord;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Optional;
+
+public enum DiscordMessageFormat {
+
+    EMBEDDED("""
+            {
+              "avatar_url": "https://www.spigotmc.org/data/resource_icons/94/94496.jpg",
+              "username": "HuskChat",
+              "content": null,
+              "embeds": [
+                {
+                  "description": "{CHAT_MESSAGE}",
+                  "color": 64410,
+                  "footer": {
+                    "text": "{SENDER_USERNAME} • {SENDER_CHANNEL}",
+                    "icon_url": "https://crafatar.com/avatars/{SENDER_UUID}?size=64"
+                  },
+                  "timestamp": "{CURRENT_TIMESTAMP}"
+                }
+              ]
+            }"""),
+
+    INLINE("""
+            {
+              "avatar_url": "https://crafatar.com/avatars/{SENDER_UUID}?size=128",
+              "username": "[{SENDER_CHANNEL}] {SENDER_USERNAME}",
+              "content": {CHAT_MESSAGE},
```

Comment:
Needs surrounding with quotations. Also todo: test if sending a quote mark breaks this

Reply:
How did I not see that when I reviewed this? 🤦 Sorry about that.

Found some other issues, I'll do the quote mark test along with the other testing I'm doing right now.

Reply:
The comma at the end of the `content` line needs to be removed for the inline DiscordMessageFormat to work.

Also, the message should be escaped. Sending a quote mark or backslash results in Discord returning HTTP 400. Escaping as follows should work:
```java
message.replace("\\", "\\\\")
             .replace("\"", "\\\"");
```